pocs/utils/images/__init__.py

```diff
@@ -112,11 +112,10 @@ def _make_pretty_from_fits(fname=None, figsize=(10, 8), dpi=150, alpha=0.2, pad=
 
     percent_value = kwargs.get('normalize_clip_percent', 99.9)
 
-    title = '{} (Exposure time: {} s, Filter: {}) {}'.format(title, exp_time, filter_type, date_time)
+    title = '{} ({}s {}) {}'.format(title, exp_time, filter_type, date_time)
     norm = ImageNormalize(interval=PercentileInterval(percent_value), stretch=LogStretch())
 
-    hdu = fits.open(fname)[0]
-    wcs = WCS(hdu.header)
+    wcs = WCS(header)
```

Comment:
Sorry, should have mentioned this before, but can you move this up to where you get the `header` and `data`? They are all kind of related ("get all the items from the fits file that are needed for plotting") so best to group them together.

Reply:
My github page got loaded weird so I think it just re-submitted this. If so, ignore.

Reply:
No problem.